test/runner/e2e_test.cpp

```diff
@@ -83,6 +87,93 @@ class EndToEndTest : public DBTest {
             datasetType == TestGroup::DatasetType::CSV_TO_JSON) {
             std::filesystem::remove_all(tempDatasetPath);
         }
+
+        if (!TestHelper::REWRITE_TESTS)
+            return;
+        std::fstream file;
+        std::string newFile;
+        std::string currLine;
+        file.open(testPath);
+
+        for (auto& statement : testStatements) {
+            while (getline(file, currLine)) {
+                if (!currLine.starts_with("-STATEMENT")) {
+                    newFile += currLine + '\n';
+                    continue;
+                }
+
+                newFile += currLine + '\n';
+                std::string stmt = currLine;
+                while (getline(file, currLine))
+                    if (currLine.starts_with("----"))
+                        break;
+                    else {
+                        newFile += currLine + '\n';
+                        stmt += currLine;
+                    }
+
+                auto normalize = [](const std::string& s) {
+                    std::string result;
+                    bool in_space = false;
+                    for (char c : s) {
+                        if (std::isspace(static_cast<unsigned char>(c))) {
+                            if (!in_space) {
+                                result += ' ';
+                                in_space = true;
+                            }
+                        } else {
+                            result += c;
+                            in_space = false;
+                        }
+                    }
+                    return result;
+                };
+
+                if (normalize(stmt) != (normalize("-STATEMENT " + statement->query))) {
+                    newFile += currLine + '\n';
+                    continue;
+                }
+
+                else {
+                    switch (statement->testResultType) {
+                    case ResultType::OK: {
+                        newFile += statement->newOutput;
+                    } break;
+                    case ResultType::HASH: {
+                        newFile += currLine + '\n';      // Add result specifier
+                        newFile += statement->newOutput; // Add produced hash
+                        getline(file, currLine);         // Ignore expected hash
+                    } break;
+                    case ResultType::TUPLES: {
+                        int skip = std::stoi(currLine.substr(5)); // Ignore expected tuples
```

Comment:
Comment is misleading. This line is extracting the existing number of tuples.

Update comment and perhaps show the expected format from which the magic number 5 is coming.

`skip` -> `count`